aws/service.go

```diff
@@ -1878,6 +1878,13 @@ func (r ConnectionErrRetryer) ShouldRetry(req *request.Request) bool {
 				if strings.Contains(awsErr.OrigErr().Error(), "http://169.254.169.254/latest") && req.RetryCount > 3 {
 					return false
 				}
+
+				// AWS Well-Architected Tool is not supported in all region yet. For unsupported regions It throws err
+				// Post "https://wellarchitected.ap-northeast-3.amazonaws.com/workloadsSummaries": dial tcp: lookup wellarchitected.ap-northeast-3.amazonaws.com: no such host
+				// https://aws.amazon.com/about-aws/whats-new/2020/04/aws-well-architected-tool-available-additional-regions/
```

Comment:
Is there a better link we can show? This is from 2020, and is bound to change over time. Is there a page that has a list of currently supported regions?